Molio/Source/Presentation/SearchUser/View/LoginRequiredView.swift

```diff
@@ -0,0 +1,37 @@
+import SwiftUI
+
+struct LoginRequiredView: View {
+    var didTabLoginRequiredButton: (() -> Void)
+    
+    var body: some View {
+        VStack {
+            Spacer()
+            Spacer()
+            Spacer()
+            Spacer()
+            Spacer()
+            Spacer()
+            Text(StringLiterals.loginRequired)
+                .font(.pretendardSemiBold(size: 24))
+                .foregroundStyle(.white)
+                .multilineTextAlignment(.center)
+            BasicButton(type: .loginRequired) {
+                didTabLoginRequiredButton()
+            }
+            .padding(.horizontal, 20)
+            Spacer()
+            Spacer()
+            Spacer()
+            Spacer()
+            Spacer()
+        }
```

Comment:
P3;
위아래 `Spacer()`가 1개씩만 있어도 될 것 같습니다!
혹시 1개씩 있을 땐 비율이 안맞았나요?
```suggestion
        VStack {
            Spacer()
            Text(StringLiterals.loginRequired)
                .font(.pretendardSemiBold(size: 24))
                .foregroundStyle(.white)
                .multilineTextAlignment(.center)
            BasicButton(type: .loginRequired) {
                didTabLoginRequiredButton()
            }
            .padding(.horizontal, 20)
            Spacer()
        }
```

Reply:
디자인상 Text가 중앙에 정렬되어있고 그 아래 버튼이 있는 형태더라구요..!!
Spacer()가 위 아래로 하나있으면 Text와 버튼이중앙으로 오는 디자인이 되는 것 같은데 그렇게 수정할까요 ?!
디자인상 6:5의 비율이길래 이렇게 작업하긴 했습니다 !!